src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private String symbol = "-";
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
```

Comment:
Comments should be indented relative to their position in the code.  
Like
public String getStatusIcon() {
        // mark done task with X
        return (isDone ? "X" : " "); 
}